### DISCUSSION TITLE
How to share chunks between processes?

### DISCUSSION THREAD

**Discussion by merwaaan** [alex8088/electron-vite#332] [Q&A]
Hi there,

Is it possible to share chunks between processes, in the sense that they are not duplicated in different build subdirectories?

Our use case:

- We rely on a native node addon (DLL file, quite large)
- Our preload process imports it (to expose its functions to the renderer process)
- Our main process imports it too (because our app has a CLI that uses the same features without UI)

This large DLL is copied to two locations on builds: out/main and out/preload.
We want to avoid that, which would halve our app's size.

Is it possible to copy it somewhere else and then import from that shared location for both processes?

**Comment by alex8088** [comment]
@merwaaan  https://electron-vite.org/guide/assets#asset-handling

**Comment by merwaaan** [comment]
@alex8088 The documentation states that such resources are not shared between main and renderer processes though: https://electron-vite.org/guide/assets#public-directory

Is there any way to make this work?
